mapadroid/mad_apk/wizard.py

```diff
@@ -280,32 +287,41 @@ def find_latest_pogo(self, architecture: APKArch) -> Optional[str]:
             if latest_pogo_version[architecture] is None:
                 return latest
             latest_supported = latest_pogo_version[architecture]
-            current_version = self.storage.get_current_version(APKType.pogo, architecture)
-            if type(current_version) is not str:
-                current_version = None
+            current_version_string = self.storage.get_current_version(APKType.pogo, architecture)
+            current_version_code = None
+            if type(current_version_string) is not str:
+                current_version_string = None
+            if current_version_string:
+                ls: dict = {architecture: current_version_string}

```

Comment:
We should use a more descriptive variable name than `ls`.

Reply:
Done. Had to add some sort of suffix because we already have a variable `latest_supported`.